src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -46,12 +47,24 @@ public ModelManager(
         this.activityBook = new ActivityBook(activityBook);
         this.internalState = new InternalState(internalState);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        context = Context.MAIN;
     }
 
     public ModelManager() {
         this(new AddressBook(), new UserPrefs(), new InternalState(), new ActivityBook());
     }
 
+    //=========== UserPrefs ====================================================
```

Comment:
This fencing seems odd - I assume this is a TODO to serialise the `Context` into JSON eventually?

Reply:
I copy pasted but forgot to change it to context oops